test/TestUtils.ts

```diff
@@ -11,9 +44,8 @@ import * as TestOverrides from './test-related-lib/TestOverrides';
  * 		expect(ctx.stdout).to.contain('No rule violations found.');
  * 	});
  */
-const setupCommandTest = test
+export const setupCommandTest = test
```

Comment:
We should decide on how we export stuff, but I made this more consistent with how other files export.